frontend/src/components/blocks/BlockDetails.tsx

```diff
@@ -17,19 +17,26 @@ export default ({ block }: Props) => {
       <Row noGutters>
         <Col className="block-info-container">
           <Row noGutters className="block-info-header">
-            <Col md="3">
+            <Col md="4">
               <CardCell
                 title="Transactions"
                 imgLink="/static/images/icon-m-transaction.svg"
                 text={block.transactionsCount.toLocaleString()}
                 className="border-0"
               />
             </Col>
-            <Col md="2">
+            <Col md="4">
               <CardCell
-                title="Size"
+                title="Gas Used"
                 imgLink="/static/images/icon-m-size.svg"
-                text="TODO"
+                text={block.gasUsed.toLocaleString()}
```

Comment:
Given it should be a string, use the same util function that we use for NEAR amounts (I hope it is generic enough).

Reply:
gasUsed is in "gas" units, which are not the same as NEAR units.
E.g. we should should just 1 Mgas for 1e6 of gas or 1 Tgas for 1e12 of gas. That is what we use currently in nearcore.

Reply:
@frol I will create a gas component since we reuse it many times